products/compute/api.yaml

```diff
@@ -146,7 +146,6 @@ objects:
           internally during updates.
         update_url: 'projects/{{project}}/regions/{{region}}/addresses/{{name}}/setLabels'
         update_verb: :POST
-        min_version: beta
```

Comment:
I don't see labels in the GA api, do you? https://cloud.google.com/compute/docs/reference/rest/v1/addresses